subscriptions/follow_up/automatic_alerts.rst

```diff
@@ -0,0 +1,89 @@
+=======================
+Set up automatic alerts
+=======================
+
+Now that your subscriptions are up and running, you want to stay up-to-date with your customers.
+Some automation would be appreciated since you would not want to go through the list of all your
+subscribers to check how things are going. This is what the *Automatic Alerts* feature is for.
+
+For example, when customers subscribe to your magazine, you would probably want to send them an
+email to welcome them and express your gratitude. Or, if the satisfaction rate of your customers
+drops below 50%, you would probably want to schedule a call with them to understand the reasons for
+their dissatisfaction.
+
+With **Odoo Subscriptions**, you can set automatic emails, you can also create a "Call" task for one
+of your salespeople so that he/she can try to understand your customer's dissatisfaction, and finally,
+why not automatically send satisfaction surveys so customers can evaluate your services? All of that
+is now possible.
```

Comment:
```suggestion
With **Odoo Subscriptions**, you can set automatic emails, create a "Call" task for one
of your salespeople so that he/she can try to understand your customer's dissatisfaction, and finally,
why not automatically send satisfaction surveys so customers can evaluate your services? All of that
is now possible.
```